src/libultra/io/devmgr.c

```diff
@@ -34,31 +35,32 @@ void __osDevMgrMain(void* arg) {
             }
 
             osRecvMesg(devMgr->acsQueue, &sp6C, OS_MESG_BLOCK);
-            __osResetGlobalIntMask(0x00100401);
-            __osEPiRawWriteIo(ioMesg->piHandle, 0x05000510, transfer->bmCtlShadow | 0x80000000);
+            __osResetGlobalIntMask(OS_IM_PI);
+            __osEPiRawWriteIo(ioMesg->piHandle, LEO_BM_CTL, transfer->bmCtlShadow | 0x80000000);
```

Comment:
```suggestion
            __osEPiRawWriteIo(ioMesg->piHandle, LEO_BM_CTL, transfer->bmCtlShadow | LEO_BM_CTL_START);
```
I think this makes sense?